kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -103,6 +103,12 @@
       type="warning">
       {{ $t('exoplatform.kudos.info.noKudosLeft', {0: numberOfKudosAllowed, 1: $t('exoplatform.kudos.label.day'), 2: kudosPeriodType}) }}
     </v-alert>
+    <v-alert
+      v-if="kudosSent"
+      dismissible
+      type="success">
+      {{ $t('exoplatform.kudos.success.kudosSent') }}
+    </v-alert>
```

Comment:
We should use the [common component](https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/components/ExoNotificationAlert.vue) in social to display the alerts.